Shared/mods/deathmatch/logic/lua/CLuaFunctionParser.h

```diff
@@ -551,6 +555,8 @@ struct CLuaFunctionParserBase
             }
             return static_cast<T>(result);
         }
+        else if constexpr (std::is_base_of_v<SColor, T>)
+            return T(static_cast<unsigned long>(lua::PopPrimitive<int64_t>(L, index)));
```

Comment:
Review note: old code is this

https://github.com/multitheftauto/mtasa-blue/blob/927509565ab3a8d48df006440654c38e0942aa89/Shared/sdk/CScriptArgReader.h#L479-L489